fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -2259,4 +2259,13 @@ public class Config extends ConfigBase {
     })
     public static boolean ignore_unknown_metadata_module = false;
 
+    @ConfField(mutable = true, masterOnly = true, description = {
+        "FE扩容时，从主节点同步元数据的timeout时间，根据${meta_dir}/image文件夹下面的image文件大小和节点间的网络环境调整，"
+            + "单位为秒，默认值300",
+        "The timeout for new FE Follower/Observer synchronizing metadata from the FE Master, "
+            + "adjust by the size of image file in the ${meta_dir}/image and the network environment between nodes. "
+            + "The default values is 300s."
+    })
+    public static int sync_new_image_timeout_s = 300;
```

Comment:
sync_image_timeout_second is a better name.

Reply:
done